src/main/java/Ex14.java

```diff
@@ -0,0 +1,31 @@
+import java.util.*;
+
+public class Ex14 {
+    public boolean armstrong(int n) {
+        int S = 0;
```

Comment:
Tên biến không đúng convention và thiếu ý nghĩa 
```suggestion
        int sum = 0;
```